extensions/mqtt/processors/PublishMQTT.cpp

```diff
@@ -75,7 +75,7 @@ void PublishMQTT::onTrigger(const std::shared_ptr<core::ProcessContext> &context
     yield();
     return;
   }
-	
+  
```

Comment:
Whitespace issues like this are not addressed in this PR, but they will be also removed in a different PR, when the relevant linter recommendation is addressed.